tests/data/policy.json

```diff
@@ -0,0 +1,7 @@
+{
+    "default": [
+        {
+            "type": "insecureAcceptAnything"
```

Comment:
It means that we will not verify the container signatures for our test purposes.